homeassistant/components/tuya/sensor.py

```diff
@@ -363,6 +363,15 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        TuyaSensorEntityDescription(
+            key=DPCode.WORK_STATE,
```

Comment:
This should be mapped to a device class so we can provide translations for these states.